### ISSUE TITLE
"Ignore cover" resets after making custom title/author

### ISSUE THREAD

**Issue by mergen3107** [koreader/koreader#12588]
* KOReader version: 2024.07-214-ga1edbbf0c
* Device: Kindle Scribe HF (FW 5.16.21)

#### Issue
`Ignore cover` (and maybe `Ignore metadata`) setting resets after I set custom metadata.

#### Steps to reproduce
- Set `Ignore cover`
- Go to Book information
- Set custom title or author
- GO back to FM/History, observe that cover is back.

##### `crash.log` (if applicable)
Pretty easy to do, but let me know if logs is needed.
(Also, I didn't find any mentions of changing custom metadata or ignore covers in the log anyway)

**Comment by hius07** [comment]
Yes, it works that way. Changing a book property creates new record for this book, the "Ignore" flags are reset.

**Comment by mergen3107** [comment]
I think it should apply the same settings again, in that case

**Comment by mergen3107** [comment]
> Changing a book property creates new record for this book, the "Ignore" flags are reset.

I really think that `book properties` should include current `ignore` flags.

**Comment by mergen3107** [comment]
These are separate things after all
